_partials/_caggs-one-step-policy.mdx

```diff
@@ -0,0 +1,27 @@
+import Experimental from "versionContent/_partials/_experimental.mdx";
+
+<h2>
+  Use a one-step policy definition to set a {props.policyType} policy on a
```

Comment:
Where will this be shown?
Until we get more feedback, we are not trying to move users to the new functions vs. the current process.

Reply:
I put it on all the how-to guides for setting refresh, compression, and retention policies. If you don't want to advertise the new functions too broadly, I can take it off the how-to guides for now and just add the API references.